npm/builder.js

```diff
@@ -174,10 +178,32 @@ Builder.prototype.runTest = function (callback) {
   callback();
 };
 
-Builder.prototype.completeRelease = function (callback) {
+Builder.prototype.pushRelease = function () {
+  var result = child_process.execSync('git remote -v').toString('utf8')
+  var lines = result.split(/\r?\n/);
+  let remoteName;
+  for (let line of lines) {
+    if (line.includes('(push)') && line.includes('asciidoctor/asciidoctor.js.git')) {
+      remoteName = line.split('\t')[0];
+      break;
+    }
+  }
+  if (typeof remoteName === 'undefined') {
+    log.warn('Unable to find the remote name of the original repository asciidoctor/asciidoctor.js');
+    return false;
+  } else {
+    this.execSync('git push ' + remoteName + ' master');
+    this.execSync('git push ' + remoteName + ' --tags');
```

Comment:
Someone has to push the commit and the tag created by `npm version` right ?
Until now, I could do it automatically because depending on the local copy the "original" repository may or may not have a remote defined.

For instance, you could have the following `git remote`:
```
origin	git@github.com:ldez/asciidoctor.js.git (fetch)
origin	git@github.com:ldez/asciidoctor.js.git (push)
```

or:
```
fork			git@github.com:ldez/asciidoctor.js.git (fetch)
fork			git@github.com:ldez/asciidoctor.js.git (push)
original	git@github.com:asciidoctor/asciidoctor.js.git (fetch)
original	git@github.com:asciidoctor/asciidoctor.js.git (push)
```